CMakeLists.txt

```diff
@@ -137,25 +138,25 @@ IF (DOWNLOAD_TITLE_SEQUENCES)
     install(CODE "file(REMOVE \$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/share/${PROJECT}/title/title-sequences.zip)")
```

Comment:
I just seem to be blind. I hope I finally got everything now, a bit of grepping didn't yield any leftovers.